iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/dispatcher/IcebergTableOperationExecutor.java

```diff
@@ -41,6 +50,34 @@ public IcebergTableOperationExecutor(IcebergCatalogWrapperManager icebergCatalog
   @Override
   public LoadTableResponse createTable(
       IcebergRequestContext context, Namespace namespace, CreateTableRequest createTableRequest) {
+    String authenticatedUser = context.userName();
+    if (StringUtils.isNotBlank(authenticatedUser)
+        && !AuthConstants.ANONYMOUS_USER.equals(authenticatedUser)) {
+      String existingOwner = createTableRequest.properties().get(OWNER_PROPERTY);
+
+      // Updated the owner as the authenticated user, if owner missing or different from
+      // authenticated user
+      if (existingOwner == null || !existingOwner.equals(authenticatedUser)) {
```

Comment:
How about using `if(!authenticatedUser.equals(existingOwner)) {}` to simple the code?